src/gpuarray/ext_cuda.h

```diff
@@ -28,10 +30,11 @@ static void setup_ext_cuda(void) {
   cuda_make_ctx = (gpucontext *(*)(CUcontext, int))gpuarray_get_extension("cuda_make_ctx");
   cuda_get_stream = (CUstream (*)(void *))gpuarray_get_extension("cuda_get_stream");
   cuda_make_buf = (gpudata *(*)(void *, CUdeviceptr, size_t))gpuarray_get_extension("cuda_make_buf");
-  cuda_get_ptr = (CUdeviceptr (*)(gpudata *))gpuarray_get_extension("cuda_get_ptr");
```

Comment:
The function this references has been removed a while ago and nothing has broken, so I quite certain it's not int use.